src/lib/expansion/expansion-panel.scss

```diff
@@ -17,13 +17,15 @@
     &:not(.mat-expanded), &:not(.mat-expansion-panel-spacing) {
       border-radius: 0;
     }
-
-    &:first-of-type {
+  
+    > &:first-child,
```

Comment:
The reason I went with `first-of-type`/`last-of-type` when I did it was because `first-child`/`last-child` can break easily if some other gets added dynamically (e.g. we have this issue on `mat-table` when a ripple is added). It might make more sense to have [something like the line setter](https://github.com/angular/material2/blob/master/src/lib/core/line/line.ts#L33) from `MatList` where we set a `first`/`last` class based on panel's index in a `QueryList`.